user/articles/3-layout-and-design/12-widget-reference.markdown

```diff
@@ -52,99 +41,77 @@ image is assigned to the category in (set in *Site Menu* &rarr;
+Custom checkout steps can be added using the 
+[custom checkout step extension point](/web/liferay-emporio/documentation/-/knowledge_base/1-0/custom-checkout-step).
 
 ## Commerce Categories Navigation
 
 The Commerce Categories Navigation widget works with [Category
 Content](#category-content) facilitate navigation to product category pages.
 
-![Figure 6: On the surface, categories navigation is just a list of links.](../../images/commerce-categories-navigation.png)
-
 Select a vocabulary on the widget's configuration screen. All of the categories
 in that vocabulary will be displayed by the widget as links to category pages.
-When a user selects one of those links, that selection is provided as context
-to the Category Content widget, which then displays the desired category.
+When a user selects one of those links, that selection is provided as context to
+the Category Content widget, which then displays the desired category.
 
 ## Orders
 
-At present, Commerce Order Content and Pending Orders are two separate widgets.
-In a future version, @commerce@ will combine them into one widget which will be
-configurable to display either past or pending orders.
-
-### Commerce Order Content
+### Orders
 
-The Order Content widget displays information about past orders: when it was
-created, who placed it, it's current status, etc. Click on an order to display
-additional information about the transaction, with an option to reorder.
+The Order Content widget displays a list of orders that have been placed. Click
+on an order to display additional information about the transaction, with an
+option to reorder.
 
-### Pending Orders
+### Open Carts
 
-The Pending Orders widget, meanwhile, displays orders that have not been
-placed. The cart's current contents are included as a pending order, as well as
+The Pending Orders widget displays orders that have not been submitted to the
+seller. The cart's current contents are included as a pending order, as well as
 any orders that are in the process of a buyer-side workflow.
 
-![Figure 7: Pending Orders is intended particularly for B2B purchasers who need to generate orders and then store them while waiting for approval.](../../images/pending-orders.png)
-
 ## Coupon Code Entry
 
 If you've created a discount that requires the entry of a coupon code, then
 buyers who know the code will need something to do with it. Drop a coupon code
 widget onto a page--sharing a page with Cart or Pending Orders is logical, but
 you can put it anywhere a buyer will have access prior to completing an order.
 
-![Figure 8: Here a buyer can enter a coupon code right before checking out.](../../images/cart-and-summary.png)
-
 See
 [Discounts](/web/liferay-emporio/documentation/-/knowledge_base/1-0/discounts)
 for more on coupon codes.
 
 ## Product Comparison Table 
 
 Displays the information for different products side-by-side. The Product
-Comparsion Bar---see below---is used to add products to the table.
```

Comment:
Every description should stand on its own and not reference the location of other descriptions within the doc.